external_dependencies.py

```diff
@@ -1,334 +1,18 @@
-"""
-CellProfiler is distributed under the GNU General Public License.
-See the accompanying file LICENSE for details.
+import os
+import urllib
 
-Copyright (c) 2003-2009 Massachusetts Institute of Technology
-Copyright (c) 2009-2015 Broad Institute
-All rights reserved.
+VERSION = "1.0.1"
 
-Please see the AUTHORS file for credits.
+prokaryote = '{}/prokaryote-{}.jar'.format(os.path.abspath("imagej/jars"), VERSION)
```

Comment:
@0x00B1 it looks like you might need to make the imagej/jars directory if it doesn't exist - mine was missing after a git clean -f -d -x